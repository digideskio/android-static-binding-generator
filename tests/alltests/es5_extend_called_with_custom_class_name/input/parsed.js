(function () {
    var MyButton = android.widget.Button.extend("my.custom.ClassName", {
        onClick1: function () {
        },
        onClick2: function () {
        }
    });
})();