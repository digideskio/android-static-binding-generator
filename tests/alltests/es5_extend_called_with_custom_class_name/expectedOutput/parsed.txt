android.widget.Button**onClick1,onClick2*my.custom.ClassName
